hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/properties/MirrorNodeEvmProperties.java

```diff
@@ -72,6 +72,9 @@ public class MirrorNodeEvmProperties implements EvmProperties {
     @NotNull
     private HederaNetwork network = HederaNetwork.TESTNET;
 
+    @Getter
+    private boolean isAllowanceEnabled = false;
```

Comment:
Lombok should generate the `is` prefix for the getter.
```suggestion
    private boolean allowanceEnabled = false;
```